README.md

```diff
@@ -1,21 +1,19 @@
 # DuckDuckGo Privacy Configuration
 
-The configuration files within this repo are used by DuckDuckGo's Apps and Browser Extensions and control which privacy protections are enabled or disabled.
+The configuration files within this repo are used by DuckDuckGo's Apps and Browser Extensions to control which privacy protections are enabled or disabled.
 
-In some cases, privacy protections can cause conflicts that affect expected website functionality. `features` can be used to temporarily disable privacy protections in order to restore expected site functionality.
- 
-This repo also contains code to build and deploy the configuration files.
+In some cases, privacy protections can cause conflicts that affect expected website functionality. Files in the `features` directory can be used to temporarily disable privacy protections or add exceptions for particular sites in order to restore expected site functionality.
+
+All feature files contain an `exceptions` property which contains a list of sites on which a feature should be disabled to prevent breaking site functionality. Additionally there is an `overrides` directory which contains files that may add to these configurations for their respective platforms.
  
-These files (in the `generated` directory) are served from: https://staticcdn.duckduckgo.com/trackerblocking/config/v1/...
+This repo also contains the code to build and deploy the configuration files.
  
-**Adding unprotected entries**
-
-To add an unprotected entry manually, update the `exceptions` of `features/unprotected-temporary.json`.
```

Comment:
I think we should keep this and further explain what it's for given it's our global opt out. Explain that we don't use it lightly etc.